### PR TITLE
Clean CSS

### PR DESCRIPTION
**Pull request by jecisc**
- Remove reference to an external resource
- Add fallback font
- Remove uses of `!important` that is not recommended in CSS
- Other small cleanings

Fixes https://github.com/ESUG/esug.github.io/issues/30

### CODE REVIEW

**Comment by SergeStinckwich** [comment]
@samakhov ?

**Comment by guillep** [comment]
No, this is almost wrong, changes should *at least* go also to pillar, otherwise we will lose them as soon as we do regenerate the site...

**Comment by jecisc** [comment]
@guillep I am not sure I understood what you mean. Can you rephrase, please? 

From what I understood I have the impression that this template is specific for the ESUG website. Am I wrong? So I do not understand why I should modify Pillar for this PR.

**Comment by guillep** [comment]
No, the template is reusable.

**Comment by jecisc** [comment]
Where is the original source?

**Comment by samakhov** [comment]
Okay, @jecisc shouldn't impact views,
but for the footer, the background image will not displayed.
I was using !important when forcing some attributes of bootstrap 3.7 for designing the navbar.

We are using ecstatic themes, but by now  have not yet packaged it to a special themes because I am using many CSS from original site esug.org.

**Comment by jecisc** [comment]
For the footer, I replaced the image that was a linear gradient with a CSS linear gradient directly. So the image should now be useless and the rendering the same.

For the `!important`, I checked and I did not see any rule overriding this one. And if there is one, in general, it's better to use a selector with a higher priority than to use `!important`. (There is explanations on the Mozilla documentation: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

**Comment by samakhov** [comment]
Cool for the footer.
I think it is Ok. It is more clear. And now I am not even using those rules.
I am using bootstrap 4.0. 

Thanks for cleaning. I will add the theme as original theme in ecstatic soon.

**Comment by guillep** [comment]
どうもありがとう アスバつ君!

So I integrate this and you backport it to pillar?
